File: nan21.dnet.module.ad.ui.extjs/src/main/resources/webapp/js/ds/JobLogMessage_Ds.js
/**
 * DNet eBusiness Suite
 * Copyright: 2010-2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
	
Ext.define(Dnet.ns.ad + "JobLogMessage_Ds", {
	extend: 'Ext.data.Model',
	
	fields: [
		{name:"messageType", type:"string"},
		{name:"message", type:"string"},
		{name:"jobLogId", type:"string"},
		{name:"id", type:"string"},
		{name:"clientId", type:"string"},
		{name:"createdAt", type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"modifiedAt", type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"createdBy", type:"string"},
		{name:"modifiedBy", type:"string"},
		{name:"notes", type:"string"},
		{name:"active", type:"boolean"},
		{name:"version", type:"int", useNull:true},
		{name:"refid", type:"string"},
		{name:"entityAlias", type:"string"},
		{name:"entityFqn", type:"string"}
	]
});

File: nan21.dnet.module.ad.presenter/src/main/java/net/nan21/dnet/module/ad/presenter/impl/scheduler/model/JobLogMessage_Ds.java
/**
 * DNet eBusiness Suite
 * Copyright: 2010-2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.ad.presenter.impl.scheduler.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;
import net.nan21.dnet.core.presenter.model.AbstractAuditableDs;
import net.nan21.dnet.module.ad.domain.impl.scheduler.JobLogMessage;

@Ds(entity = JobLogMessage.class)
public class JobLogMessage_Ds extends AbstractAuditableDs<JobLogMessage> {

	public static final String f_messageType = "messageType";
	public static final String f_message = "message";
	public static final String f_jobLogId = "jobLogId";

	@DsField
	private String messageType;

	@DsField
	private String message;

	@DsField(join = "left", path = "jobLog.id")
	private String jobLogId;

	public JobLogMessage_Ds() {
		super();
	}

	public JobLogMessage_Ds(JobLogMessage e) {
		super(e);
	}

	public String getMessageType() {
		return this.messageType;
	}

	public void setMessageType(String messageType) {
		this.messageType = messageType;
	}

	public String getMessage() {
		return this.message;
	}

	public void setMessage(String message) {
		this.message = message;
	}

	public String getJobLogId() {
		return this.jobLogId;
	}

	public void setJobLogId(String jobLogId) {
		this.jobLogId = jobLogId;
	}
}
